ws/websocket.go

```diff
@@ -221,19 +237,24 @@ func (server *Server) Start(port int, listenPath string) {
 	server.httpServer.Handler = router
 	if server.tlsCertificatePath != "" && server.tlsCertificateKey != "" {
 		if err := server.httpServer.ListenAndServeTLS(server.tlsCertificatePath, server.tlsCertificateKey); err != http.ErrServerClosed {
-			log.Errorf("websocket server error: %v", err)
+			server.error(fmt.Errorf("failed to listen: %w", err))
```

Comment:
These statements are failing the CI with `ws/websocket.go:240: Errorf format %w has unknown verb w`. Check the [CI job](https://travis-ci.org/github/lorenzodonini/ocpp-go/jobs/742718154).

CI runs on Go 1.12. Is the `%w` a go 1.13-1.15 feature? If you can confirm this, I will bump the Golang version on CI.

Reply:
Thats 1.13: https://golang.org/doc/go1.13#error_wrapping

Reply:
I just updated the base project. Feel free to rebase on master, then the CI should run just fine.